File: lab-web-based-java-programming/practice-evaluation/src/main/java/com/app/pojos/Course.java
package com.app.pojos;

import java.util.ArrayList;
import java.util.List;

import javax.persistence.CascadeType;
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.OneToMany;
import javax.persistence.Table;

@Entity
@Table(name = "courses")
public class Course {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	@Column(name = "course_id")
	private Integer courseId;
	@Column(length = 20, name = "course_name")
	private String courseName;
	@OneToMany(mappedBy = "course", cascade = CascadeType.ALL, orphanRemoval = true)
	@Column(name = "enrolled_students")
	private List<Student> enrolledStudents = new ArrayList<>();

	public Course() {
		super();
	}

	public Course(String courseName) {
		super();
		this.courseName = courseName;
	}

	public Integer getCourseId() {
		return courseId;
	}

	public void setCourseId(Integer courseId) {
		this.courseId = courseId;
	}

	public String getCourseName() {
		return courseName;
	}

	public void setCourseName(String courseName) {
		this.courseName = courseName;
	}

	public List<Student> getEnrolledStudents() {
		return enrolledStudents;
	}

	public void setEnrolledStudents(List<Student> enrolledStudents) {
		this.enrolledStudents = enrolledStudents;
	}

	public void addStudent(Student student) {
		enrolledStudents.add(student);
		student.setCourse(this);
	}

	public void removeStudent(Student student) {
		enrolledStudents.remove(student);
		student.setCourse(null);
	}

	@Override
	public String toString() {
		return "Course [courseId=" + courseId + ", courseName=" + courseName + ", enrolledStudents=" + enrolledStudents
				+ "]";
	}
}

File: practice-cpp-programming-language/practice-sunbeam-admission-process/core/Course.cpp
#include <iostream>

using namespace std;

class Course
{
private:
  string courseName;
  double fees;

public:
  Course(string courseName, double fees)
  {
    this->courseName = courseName;
    this->fees = fees;
  }

  string getCourseName()
  {
    return this->courseName;
  }

  double getFees()
  {
    return this->fees;
  }

  void setCourseName(string courseName)
  {
    this->courseName = courseName;
  }

  void setFees(double fees)
  {
    this->fees = fees;
  }

  void toString() 
  {
    cout << "Course [" << this->getCourseName() << ", " << this->getFees() << "]" << endl;
  }
};